qiita_db/data.py

```diff
@@ -1144,6 +1144,21 @@ def study(self):
             "processed_data_id=%s".format(self._study_processed_table),
             [self._id])[0]
 
+    @property
+    def samples(self):
```

Comment:
Change this function to make use of a single SQL query. Right now is executing at least 4.